diff -up ixion-0.16.1-def5cc4b6c7d846b05714e7495eddf0346ec401c/src/libixion/model_context.cpp.omv~ ixion-0.16.1-def5cc4b6c7d846b05714e7495eddf0346ec401c/src/libixion/model_context.cpp
--- ixion-0.16.1-def5cc4b6c7d846b05714e7495eddf0346ec401c/src/libixion/model_context.cpp.omv~	2020-10-27 21:41:07.251625437 +0100
+++ ixion-0.16.1-def5cc4b6c7d846b05714e7495eddf0346ec401c/src/libixion/model_context.cpp	2020-10-27 21:41:14.407703385 +0100
@@ -17,7 +17,7 @@
 
 namespace ixion {
 
-model_context::input_cell::input_cell(nullptr_t) : type(celltype_t::empty) {}
+model_context::input_cell::input_cell(std::nullptr_t) : type(celltype_t::empty) {}
 model_context::input_cell::input_cell(bool b) : type(celltype_t::boolean)
 {
     value.boolean = b;
